local/compose/ps.go

```diff
@@ -61,6 +63,7 @@ func (s *composeService) Ps(ctx context.Context, projectName string, options com
 				health = inspect.State.Health.Status
 			}
 
+			l.Lock()
```

Comment:
not necessary AFAIK as we pre-allocated the slice and each gortouine will run with a distinct index